data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -240,22 +242,21 @@ def _create_tile_data(
     def _create_downloadable_data(
         self, score_county_state_merged_df: pd.DataFrame
     ) -> pd.DataFrame:
-        df = score_county_state_merged_df[constants.DOWNLOADABLE_SCORE_COLUMNS]
-
-        float_columns = df.select_dtypes(include=["float64"])
-
-        # score_tiles[constants.TILES_SCORE_FLOAT_COLUMNS] = score_tiles[
-        #     constants.TILES_SCORE_FLOAT_COLUMNS
-        # ].apply(
-        #     func=lambda series: floor_series(
-        #         series=series,
-        #         number_of_decimals=constants.TILES_ROUND_NUM_DECIMALS,
-        #     ),
-        #     axis=0,
-        # )
-
-        # [x for x in df.columns if field_names.PERCENTILE_FIELD_SUFFIX in x]
-
+        df = score_county_state_merged_df[
+            constants.DOWNLOADABLE_SCORE_COLUMNS
+        ].copy(deep=True)
+
+        df_of_float_columns = df.select_dtypes(include=["float64"])
```

Comment:
I was thinking about this - we should just reference `df`'s columns as we verified that the input are float columns - although the nullable types causes indirection.